src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+
+    public String getRandomColor(){
```

Comment:
The method `getRandomColor` returns a `String`, but it might be more appropriate to return a `Color` type directly, as it would better align with the use of the `Color` enum. Consider changing the return type to `Color` and adjusting the method accordingly.